### PR TITLE
added transaction, transfer, invitation and user endpoints

### PR DESCRIPTION
**Pull request by ermiyas111**

### CODE REVIEW

**Review by suraffy — COMMENTED** [review]
👍

File: lib/invitation.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const apiRequest = require('./apiRequest');
+
+async function findByInviter() { 
+  const apiResponse = await apiRequest("GET", "/invitation/find-by-inviter", "", null)
+  return apiResponse;
+}
+
+async function createInvitation(country, phone, amount) { 
+  // {\n\t\"country\": \"Italy\",\n\t\"phone\": \"379189****\",\n\t\"amount\": 5000\n}"
```
Comment (agebeyaw) [inline lib/invitation.js:9]: you forgot to remove commented code

File: lib/transaction.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const apiRequest = require('./apiRequest');
+
+async function getTransactionListByUser() { 
+  const apiResponse = await apiRequest("GET", "/transaction/find-by-user", "?p=1", null)
+  return apiResponse;
+}
+
+async function createTransaction(receiver, amount, cause, meta_data) { 
+  // {\n\t\"receiver\": \"************\",\n\t\"amount\": 5000,\n\t\"cause\": \"\",\n\t\"meta_data\": []\n}
```
Comment (agebeyaw) [inline lib/transaction.js:9]: use block comment if you want to indicate the response format in an example, or remove it at all. There will be enough API documentation to explain the request and response formats

**Review by agebeyaw — COMMENTED** [review]
@ermiyas111 make sure you get your PR reviewed and approved before you merge.

**Comment by ermiyas111** [comment]
> @ermiyas111 make sure you get your PR reviewed and approved before you merge.

Actually I didn't merge this PR. For the other cases alright.

File: lib/invitation.js (hunk 1 above)
Comment (ermiyas111) [reply lib/invitation.js:9]: @agebeyaw  I added them on purpose to provide a template for the request object, if users want to check it. Should I remove them?

File: lib/transaction.js (hunk 2 above)
Comment (ermiyas111) [reply lib/transaction.js:9]: Will remove them

**Pull request merged**